app/grants/views.py

```diff
@@ -1593,6 +1593,9 @@ def grant_edit(request, grant_id):
         if not description_rich:
             description_rich = description
 
+        is_clr_eligible = json.loads(request.POST.get('is_clr_eligible', 'true'))
+        grant.is_clr_eligible = is_clr_eligible
```

Comment:
Can we wrap this with `if grant.active:`?

Reply:
hmm well if a grant is not active -> this endpoint shouldn't be accessible via UI.
So ideally they would not be able to edit their grants right ?

Reply:
changed it anyways to be safer